src/Altinn.Platform/Altinn.Platform.Authorization/Authorization/Migration/v0.02/02-setup-procedures.sql

```diff
@@ -0,0 +1,27 @@
+-- Procecure: insert_change
+CREATE OR REPLACE PROCEDURE delegation.insert_change(
+	_altinnappid character varying,
+	_offeredbypartyid integer,
+	_coveredbyuserid integer,
+	_coveredbypartyid integer,
+	_performinguserid integer,
+	_blobstoragepolicypath character varying,
+	_blobstorageversionid character varying,
+	_isDeleted boolean,
+	INOUT _policychangeid bigint)
+LANGUAGE 'plpgsql'
+AS $BODY$
+BEGIN
+INSERT INTO delegation.delegatedPolicy(
```

Comment:
delegatedPolicy tabellen ble renamet til delegationChanges (og policyChangeId til delegationChangeId ) under PR review perioden av min endring for AddRules.
Jeg oppdaterte da også for å legge til lagring av _isDeleted som vel var det som manglet her som du ønsket å oppdatere for?

Ser hvertfall ellers ingen endring her så tror ikke denne skal være nødvendig.